pkg/config/centralconfig.go

```diff
@@ -565,6 +576,16 @@ func (c *CentralConfiguration) validateTraceabilityAgentConfig() {
 		exception.Throw(ErrBadConfig.FormatError(pathEventAggregationInterval))
 	}
 
+	// On validation if Event Aggregation is in Offline set the interval to at most hourly
```

Comment:
isn't this: to at _least_ hourly?

Reply:
I am reading it as a frequency, at most it will run every hourly, is my mind just backwards on this?